### ISSUE TITLE
Time Filter - 1.3.1 Info and Relationships - The title of the panel does not match the screen reader interpretation

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#10243]
As a user reliant on a screen reader, I expect all screen reader label text to exactly match the text visually displayed on the page. This helps users with learning disabilities or dyslexia to understand text with the aid of screen readers.

**Linked work item:**
#9500 

**Expected Result:**
The textual title of the panel (visually represented on the page) should be 'Time Filter' to match the screen reader interpretation. 

**Actual Result:**
The title of the panel is Date Interval, but the screen reader identifies the title as 'Time Filter' when the user focuses on this component, which will confuse some users. 

**Evidence:**

https://github.com/archesproject/arches/assets/77161085/8aced0af-80a8-40f2-8af0-a7ebc287dd32

**Comment by johnatawnclementawn** [comment]
@imogenroberts, please pull the latest from the accessibility branch and retest

**Comment by imogenroberts** [comment]
RETEST: Panel's 'close' button now has no title visually associated with it which avoids any confusion about the title of the panel for screen reader users. 

**Evidence:**


![Image](https://github.com/archesproject/arches/assets/77161085/522aa29e-412d-4fa5-a445-3cfa39c5fc59)

**Comment by imogenroberts** [comment]
RETEST AMENDMENT: Time Filter title appears on the panel!

**Evidence:**


![Image](https://github.com/archesproject/arches/assets/77161085/330b06eb-3b06-40c3-a44f-7647781e5d1c)
